### PR TITLE
Harden M::B code to be strict compliant in tests and Build.PL generat…

### PR DESCRIPTION
**Pull request by toddr**
…ion.

### CODE REVIEW

File: lib/Module/Build/Base.pm [hunk 1]
```diff
@@ -1079,8 +1079,10 @@ sub subclass {
   open(my $fh, '>', $filename) or die "Can't create $filename: $!";
   print $fh <<EOF;
 package $opts{class};
+use strict;
```
Comment (Leont) [inline lib/Module/Build/Base.pm:1082]: This will break if the subclass code was not in fact strict.

Comment (toddr) [reply lib/Module/Build/Base.pm:1082]: I'm happy to take that out. My goal is the our statements not the strict.

Comment (Grinnz) [reply lib/Module/Build/Base.pm:1082]: Agreed, adding just the ours should be fine.

Comment (toddr) [reply lib/Module/Build/Base.pm]: done.

**Review by Leont — APPROVED** [review]

**Pull request merged**